application/app/Http/Resources/ClassifierValueResource.php

```diff
@@ -0,0 +1,41 @@
+<?php
+
+namespace App\Http\Resources;
+
+use App\DataTransferObjects\Meta\LanguageMetaData;
+use App\DataTransferObjects\Meta\ProjectTypeMetaData;
+use App\Enums\ClassifierValueType;
+use App\Models\ClassifierValue;
+use Illuminate\Http\Request;
+use Illuminate\Http\Resources\Json\JsonResource;
+
+/**
+ * @mixin ClassifierValue
+ */
+class ClassifierValueResource extends JsonResource
+{
+    /**
+     * Transform the resource into an array.
+     *
+     * @return array<string, mixed>
+     */
+    public function toArray(Request $request): array
+    {
+        return [
+            'id' => $this->id,
+            'name' => $this->name,
+            'value' => $this->value,
+            'type' => $this->type->value,
```

Comment:
**[non-blocker]** Just pointing out that it seems Laravel automatically converts backed enums to their backing value, so the following seems to be functionally equivalent:

```suggestion
            'type' => $this->type,
```